test/fixtures/lightscript/arrow-methods/class-malformed-async-skinny-arrow/actual.js

```diff
@@ -0,0 +1,3 @@
+class X {
+  async method() -> await a
```

Comment:
Alright, I'm adding a new, separate test for that so we'll keep this one. Also to specify - a getter can return a Promise, it just can't be an actual `async` function